db/seeds.d/103-provisioning_templates.rb

```diff
@@ -22,7 +20,15 @@
 
 templates.each do |template|
   template[:template] = File.read(File.join(Katello::Engine.root, "app/views/foreman/unattended", template.delete(:source)))
-  ProvisioningTemplate.find_or_create_by_name(template).update_attributes(defaults.merge(template))
+  ProvisioningTemplate.where(:name => template["name"]).first_or_create do |pt|
+    pt.vendor = "Katello"
+    pt.default = true
+    pt.locked = true
+    pt.name = template[:name]
+    pt.template = template[:template]
+    pt.template_kind = template[:template_kind] if template[:template_kind]
+    pt.snippet = template[:snippet] if template[:snippet]
```

Comment:
Why was this change needed? All of these attributes are attr_accessible:

https://github.com/theforeman/foreman/blob/develop/app/models/template.rb#L4-L5